packages/wrangler/src/dev/dev.tsx

```diff
@@ -55,6 +55,8 @@ export type DevProps = {
 	zone: string | undefined;
 	host: string | undefined;
 	routes: Route[] | undefined;
+	isApi?: boolean;
```

Comment:
wondering whether "isApi", should actually be params for `logLevel` and `inspect`